### PR TITLE
Add granular tax properties

### PR DESCRIPTION
**Pull request by VladimirBerdnik**

### CODE REVIEW

File: openapi/components/schemas/Invoices/InvoiceTaxItem.yaml [hunk 1]
```diff
@@ -16,3 +16,61 @@ properties:
...
+    nullable: true
+    example: 0.38
+  jurisdictions:
+    description: Jurisdiction names for the invoice.
+    type: object
+    nullable: true
+    readOnly: true
+    properties:
+      country:
+        description: Two-letter ISO country code for given location.
```
Comment (noon-dawg) [inline openapi/components/schemas/Invoices/InvoiceTaxItem.yaml:54]: nitpick:
```suggestion
        description: Two-letter ISO country code for the given location.
```

File: openapi/components/schemas/Invoices/InvoiceTaxItem.yaml [hunk 2]
```diff
@@ -16,3 +16,61 @@ properties:
...
+        type: string
+        nullable: true
+        readOnly: true
+        example: US
+      state:
+        description: Postal abbreviated state name for given location.
```
Comment (noon-dawg) [inline openapi/components/schemas/Invoices/InvoiceTaxItem.yaml:60]: nitpick:
```suggestion
        description: Postal abbreviated state name for the given location.
```

File: openapi/components/schemas/Invoices/InvoiceTaxItem.yaml [hunk 3]
```diff
@@ -16,3 +16,61 @@ properties:
...
+        type: string
+        nullable: true
+        readOnly: true
+        example: CA
+      county:
+        description: County name for given location.
```
Comment (noon-dawg) [inline openapi/components/schemas/Invoices/InvoiceTaxItem.yaml:66]:
```suggestion
        description: County name for the provided location.
```

**Comment by VladimirBerdnik** [comment]
devs please review

**Comment by VladimirBerdnik** [comment]
devs please review

**Review by n-fasano — APPROVED** [review]

**Review by noon-dawg — APPROVED** [review]
LGTM

**Review by ashkarpetin — APPROVED** [review]

**Review by nanobash — APPROVED** [review]

**Pull request merged**